toolchain/sem_ir/formatter.cpp

```diff
@@ -1204,7 +1180,7 @@ auto Formatter::FormatInstRhs(Inst inst) -> void {
     case CARBON_KIND(ReturnExpr ret): {
       FormatArgs(ret.expr_id);
       if (ret.dest_id.has_value()) {
-        FormatReturnSlotArg(ret.dest_id);
+        FormatArgs(ret.dest_id);
       }
```

Comment:
`FormatArgs` is intended to only be called once for an instruction:
```suggestion
    case CARBON_KIND(ReturnExpr ret): {
      if (ret.dest_id.has_value()) {
        FormatArgs(ret.expr_id, ret.dest_id);
      } else {
        FormatArgs(ret.expr_id);
      }
```